{"name": "test_login_functionality_001", "status": "passed", "attachments": [{"name": "log", "source": "87776dc2-717e-4986-a302-1346a4008598-attachment.txt", "type": "text/plain"}], "start": 1743767327646, "stop": 1743767333575, "uuid": "7ec6b5be-6c1a-48c3-a448-7fbab67f4d45", "historyId": "6106e98421f5d994cbc4783840ff5923", "testCaseId": "6106e98421f5d994cbc4783840ff5923", "fullName": "test_login.Test_002_Login_Functionality#test_login_functionality_001", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_002_Login_Functionality"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "26232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}